src/dotnet/APIView/APIViewWeb/Languages/LanguageProcessor.cs

```diff
@@ -67,7 +67,15 @@ public override async Task<CodeFile> GetCodeFileAsync(string originalName, Strea
             }
             finally
             {
-                Directory.Delete(tempDirectory, true);
+                await Task.Delay(1000);
```

Comment:
is there a race condition that cause us to wait?

Reply:
Not race condition, but delay in file system releasing the resource.

Reply:
talked offline